README.md

```diff
@@ -64,6 +64,17 @@ Once skeletor has successfully been added to the build we need to include scaffo
 ./vendor/bin/skeletor-init-project.sh
 ```
 
+In order to create a sub profile based on Skeletor, you have to apply a patch to Drupal core (until it's incorporated into core). To do so, add the following to your ```composer.json```
```

Comment:
I would add the link to the issue at the end of hte parentheses. Otherwise I think this is fine.